### PR TITLE
doc: add security doc

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

no docs as reported in #3549 

**What is the chosen solution to this problem?**

add one

**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+We use changeset to provide new version everyday with fixes, features and major release when we have a breaking change.
+Let latest is the better. We also provide [talend-scripts upgrade:deps](./tools/upgrade-deps) scripts to make it easier to be aligned and updated.
+The script comes with option about security use case so please read it first.
+
+Distance to latest from npm is added on [dependencies.txt](./dependencies.txt) file on each upgrade PR.
```
Comment (Benrajalu) [inline SECURITY.md:9]:
```suggestion
Distance to latest from npm is added on a [dependencies.txt](./dependencies.txt) file on each upgrade PR.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Reporting a Vulnerability
+
+Please open an issue on github repository if you are outside Talend or in the companie tracker of your team if you are inside Talend.
+You can then ask for help if you need to address the issue.
```
Comment (Benrajalu) [inline SECURITY.md:14]:
```suggestion
Please open an issue on this github repository if you are outside Talend, or in the company's tracker for your team if you are inside Talend.

You can then ask for help if you need to address the issue.
```

Comment (Benrajalu) [reply SECURITY.md:14]: Which make me think packages should have codeowners, making "asking for help" (and reviews) easier.

**Review by Benrajalu — APPROVED** [review]

File: SECURITY.md (hunk 2 above)
Comment (jmfrancois) [reply SECURITY.md]: This is interesting proposal I would like to see.
> With protected branches enabled, a code owner for each owned file has to leave a review before anyone can merge a pull request to that branch.
But today it is quite hard to have reviews so I am afraid it makes it more complex.
I would love to try, please go ahead with a PR.

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+We also provide [talend-scripts upgrade:deps](./tools/upgrade-deps) scripts to make it easier to keep our packages and apps aligned and updated.
+
+The script comes with options about security use cases, so please read the documentation first.
+
+Distance to latest from npm is added on a [dependencies.txt](./dependencies.txt) file on each upgrade PR.
+
+## Reporting a Vulnerability
+
+Please open an issue on this github repository if you are outside Talend, or in the company's tracker for your team if you are inside Talend.
```
Comment (coheigea) [inline SECURITY.md:17]: Could this section be changed to instead reference our vulnerability disclosure page?

**Review by coheigea — CHANGES_REQUESTED** [review]

**Review by coheigea — APPROVED** [review]

**Pull request merged**